### PR TITLE
added email best practices and updated email specific doc filtering

### PR DESCRIPTION
**Pull request by CrystalOnScript**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+#Other Gotchas
+
+When working AMP for Email, keep in mind the following tips and tricks:
+
+*   The AMP for Email playground doesn't proxy XHRs, but some email providers do.
+*   The AMP MIME part should appear before the HTML MIME part in your email to ensure maximum compatibility across email clients.
+*   The src attribute of amp-list, action-xhr of amp-form, the src for amp-img, or the href attribute of an `<a>` tag cannot be mutated by amp-bind.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md]: nit: ```s/src/`src`/```

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md]: same for amp-img, amp-list, action-xhr of amp-form

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md]: and href

Comment (matthiasrohmer) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md]: >same for amp-img, amp-list, action-xhr of amp-form

In existing documents we additionally linked component names to their reference, would be cool to continue this 🙂

Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_email_best_practices.md]: Good catches/suggestions. Done!

**Pull request merged**